src/index.html

```diff
@@ -4,14 +4,148 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="sname">
+            Surname:
+          <input
+            type="text"
+            name="sname"
+            id="sname"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label for="name1">Name:
+          <input
+            type="text"
+            name="name1"
+            id="name1"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label for="years">How old are You?
+          <input type="number"
+            name="years"
+            id="years"
+            value="12"
+            min="12"
+            max="100">
+        </label>
+        <br>
+        <label for="birth">Full date of birth:
+          <input type="date"
+            name="birth"
+            id="birth">
+        </label>
+        <br>
+        <label for="agreement">I accept the term of the agreement
+          <input type="checkbox"
+            name="agreement"
+            id="agreement">
+        </label>
+        <br>
+      </fieldset>
+      <br>
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="mail">E-mail:
+          <input type="email"
```

Comment:
```suggestion
        <label for="mail">
          E-mail:
          <input type="email"
```
Переноси текст на новую строку